src/components/WorkPoolQueueStatusBadge.vue

```diff
@@ -1,70 +1,29 @@
 <template>
-  <template v-if="workQueue && workQueueStatus">
-    <p-tag class="work-pool-queue-status-badge" :class="classes">
-      <WorkPoolQueueStatusIcon v-if="can.access.workQueueStatus" :work-pool-queue="workQueue" />
-      <WorkPoolQueueHealthIcon v-else :work-queue-name="workQueueName" :work-pool-name="workPoolName" class="work-pool-queue-status-badge__icon" />
+  <template v-if="workQueue">
+    <p-tag class="work-pool-queue-status-badge">
+      <WorkPoolQueueStatusIcon :work-pool-queue="workQueue" />
       {{ label }}
     </p-tag>
   </template>
 </template>
 
 <script lang="ts" setup>
   import { computed } from 'vue'
-  import { WorkPoolQueueHealthIcon } from '@/components'
   import WorkPoolQueueStatusIcon from '@/components/WorkPoolQueueStatusIcon.vue'
-  import { useCan, useWorkQueueStatus } from '@/compositions'
-  import { WorkPool, WorkPoolQueue } from '@/models'
+  import { WorkPoolQueue } from '@/models'
 
   const props = defineProps<{
     workQueue: WorkPoolQueue,
-    workPool: WorkPool,
   }>()
 
-  const can = useCan()
-
-  const workQueueName = computed(() => props.workQueue.name)
-  const workPoolName = computed(() => props.workPool.name)
-  const workQueueId = computed(() => props.workQueue.id)
-  const { workQueueStatus } = useWorkQueueStatus(workQueueId)
-  const healthy = computed(() => workQueueStatus.value?.healthy)
-
-  const label = computed(() => {
-    if (can.access.workQueueStatus) {
-      return { 'ready': 'Ready', 'not_ready': 'Not Ready', 'paused': 'Paused' }[props.workQueue.status]
-    }
-
-    if (props.workQueue.isPaused) {
-      return 'Paused'
-    }
-    return healthy.value ? 'Healthy' : 'Unhealthy'
-  })
-
-  const classes = computed(() => `work-queue-status-badge--${label.value.toLowerCase()}`)
+  const label = computed(() => (
+    { 'ready': 'Ready', 'not_ready': 'Not Ready', 'paused': 'Paused' }[props.workQueue.status]
+  ))
 </script>
 
 <style>
 .work-pool-queue-status-badge { @apply
   text-xs
   pl-2
 }
-
-.p-tag.work-pool-queue-status-badge--healthy { @apply
```

Comment:
So the visual indicator these styles gave before is coming from the status icon alone now?

Reply:
yup!

https://github.com/PrefectHQ/prefect-ui-library/blob/e59fd70c91245fcddcd470feca59b09ce4ff711e/src/components/WorkPoolQueueStatusIcon.vue#L1

Reply:
thank you!